### ISSUE TITLE
Global ids from objects with composite primary keys are not correctly extracted

### ISSUE THREAD

**Issue by eoma** [graphql-python/graphene-sqlalchemy#43]
Due to the way graphene-sqlalchemy constructs global ids, it is not possible to get unique global ids from objects with composite primary keys, typically association objects.

**Comment by eoma** [comment]
A serialization format of complex ids should probably be decided upon too, due to the way sqlalchemy's Query.get() expects ids to be entered.

**Comment by eoma** [comment]
A somewhat simple oneway solution is to make SQLAlchemyObjectType.resolve_id return a tuple of the primary keys if it is a list with more than one entry:
```python
    def resolve_id(self, args, context, info):
        graphene_type = info.parent_type.graphene_type
        if is_node(graphene_type):
            keys = self.__mapper__.primary_key_from_instance(self)
            return tuple(keys) if len(keys) > 1 else keys[0]
        return getattr(self, graphene_type._meta.id, None)
```

**Comment by syrusakbary** [comment]
This is now merged, and also fixed in upcoming Graphene version `2.0`
